### ISSUE TITLE
some humble installation problems

### ISSUE THREAD

**Issue by funky-gibbon** [ohmyzsh/ohmyzsh#9020]
Dear All,
I am running gentoo stable on an FX 8350 box.
on boot up and log in I see the following error as a user at an LXterminal:

home/mikef/.zshrc:53: unmatched "
bucketlist% 

Zsh seems to have installed OK and runs well I think.

My .zshrc file looks like this:
```
���  mikef more .zshrc
source "`/zsh-theme-powerlevel9k/powerlevel9k.zsh-theme"

plugins=(
    git
    bundler
    dotenv
    osx
    rake
    rbenv
    ruby
)


ZSH_THEME="robbyrussell"

#ZSH_THEME="powerlevel10k/powerlevel10k"

zstyle ':completion::complete:*' use-cache 1

setopt APPEND_HISTORY
setopt INC_APPEND_HISTORY
setopt HIST_IGNORE_DUPS
setopt HIST_IGNORE_SPACE
# dont put commands beginning with space in history
setopt interactivecomments  # allows end of command line comments

setopt extendedglob
setopt extended_glob
setopt no_case_glob
# allow ls **.c instead of **/*.c
setopt GLOBSTARSHORT
setopt autopushd pushdignoredups autocd

#!/bin/zsh

#Completion
autoload -U compinit
compinit

# Correction

setopt correctall


#Prompt

autoload -U promptinit
promptinit
prompt gentoo

alias myip="curl http://ipecho.net/plain; echo"

���  mikef  
```
I can't get either powerlevel 9k or 10k to run in a terminal but both did install correctly from git.
pk10 configure is not recognised as a command even after running exec zsh

I am new to zsh.

Suggestions on improving things a bit would be appreciated.   I will look through the issue list and see if I can find any relevant posts.

Regards

Michael Fothergill

**Comment by mcornella** [comment]
> source "`/zsh-theme-powerlevel9k/powerlevel9k.zsh-theme"

There's a typo in this line, you need to remove the backtick (<code>`</code>) and fix the powerlevel9k path if you want it to work. But just removing the backtick will fix the `unmatched "` error.

**Comment by funky-gibbon** [comment]
A great many thanks for this.  I removed the ' character from the .zshrc file and it got rid of the error message as you predicted.

I then found the `powerlevel9k` directory with the` powerlevel9k-theme` file in it.

I then changed the path to `/root/powerlevel9k/powerlevel9k-theme `and opened a new terminal window.

I got a whole load of complaints about file permission problems.

So I ran chown mikef powerlevel9k and chgrp mikef powerlevel9k for that directory and for /root directly itself plus the theme file as well.

I then opened a konsole terminal window as a user (not root) and got a set of error messages saying things like` left prompt segment 33: command not found: print icon etc and left prompt end:2 command not found print icon and  left prompt end;6 command not found print icon and finally several right promp segment:48` type errors 

I will google around and see if I can't find any reference to these errors myself and try to fix them.

A little blue box with an ampersand character has now appeared next to the prompt so I think that powerlevel9k is trying to work a bit.

Regards

MF
